sdk/tables/Azure.Data.Tables/tests/TableClientLiveTests.cs

```diff
@@ -1286,6 +1286,24 @@ public async Task GetEntityIfExistsContainsValueWhenExists()
             Assert.That(entityResults, Is.Not.Null, "The entity should not be null.");
         }
 
+        [RecordedTest]
+        public async Task GetEntityAllowsEmptyRowKey()
```

Comment:
Shouldn't the empty partition key scenario also be tested to prevent regression?